src/resmom/mom_comm.c

```diff
@@ -278,8 +315,26 @@ task_save(pbs_task *ptask)
 			return (-1);
 		}
 	}
+	obitent *pobit = (obitent *)GET_NEXT(ptask->ti_obits);
```

Comment:
I am not sure but ptask->ti_obits is list, and we are only storing first? don't we need to store others in list?

Reply:
You're probably right, I thought that obits are only created when MS first sends a task, but now I see they can be created in other ways as well. I'll rethink how the obits are saved, thanks for the catch!